### ISSUE TITLE
Add movie/series titles below or top of the Picture 

### ISSUE THREAD

**Issue by Avaritia06** [Stremio/stremio-features#816]
**Stremio Version**
Stremio Version 4.4.168 and Windows Desktop

**Is your feature request related to a problem? Please describe.**
hard to search or navigate movies/series base on cover alone, because there are times the cover suddenly changes to a new cover please see below pictures

**Describe the solution you'd like**
show titles in the library like in the home button

HOME PAGE (Titles are showned)
![Screenshot 2024-04-20 000233](https://github.com/Stremio/stremio-features/assets/81761229/3fbd9734-b6a8-4689-9b6c-bdbeea314db3)

LIBRARY PAGE (No Titles are showned)
![Screenshot 2024-04-20 000214](https://github.com/Stremio/stremio-features/assets/81761229/ffcd840a-e09c-440c-8450-d0aba77df830)

**Comment by VictorVow** [comment]
Yeah pretty annoying on the Discover page not being able to see the actual titles.

**Comment by Avaritia06** [comment]
but in web.stremio.com all titles are shown either in discovery, library or home this is what is lacking in desktop annd android version

**Comment by Avaritia06** [comment]
> Yeah pretty annoying on the Discover page not being able to see the actual titles.

but in web.stremio.com all titles are shown either in discovery, library or home this is what is lacking in desktop and android version

**Comment by legndery** [comment]
any plugins? otherwise I am tempted to create one

**Comment by shaneaflyz** [comment]
I have found out that the beta version 5.0 has resolved this issue
try checking it out

search:
<img width="1300" height="433" alt="Image" src="https://github.com/user-attachments/assets/f9b23241-5fe8-4243-b357-ff3c99fa4c6e" />

library:

<img width="1214" height="599" alt="Image" src="https://github.com/user-attachments/assets/a51c37ed-054f-41e7-a66a-3c54b85154f1" />

version:
<img width="470" height="267" alt="Image" src="https://github.com/user-attachments/assets/9b0691d0-9200-4478-84bf-876ff79d0c15" />

**Comment by kenmai9** [comment]
> I have found out that the beta version 5.0 has resolved this issue try checking it out
> 
> search: <img alt="Image" width="1300" height="433" src="https://private-user-images.githubusercontent.com/136676564/471513714-f9b23241-5fe8-4243-b357-ff3c99fa4c6e.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NjE1Nzc5MTUsIm5iZiI6MTc2MTU3NzYxNSwicGF0aCI6Ii8xMzY2NzY1NjQvNDcxNTEzNzE0LWY5YjIzMjQxLTVmZTgtNDI0My1iMzU3LWZmM2M5OWZhNGM2ZS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUxMDI3JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MTAyN1QxNTA2NTVaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT02Yjg5NjUxYzU2OTZkYTkyYzNkMDdjZWJhOWQyNWEwZWJlZDk3YTIyZmE1YjczYTUwYmJlYWQxODZiMzEzMDQ5JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.Dk7tXv3BIJmCZ9fZNoHZN7qaJQ-lUgxHDYtRxWGKz94">
> 
> library:
> 
> <img alt="Image" width="1214" height="599" src="https://private-user-images.githubusercontent.com/136676564/471513967-a51c37ed-054f-41e7-a66a-3c54b85154f1.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NjE1Nzc5MTUsIm5iZiI6MTc2MTU3NzYxNSwicGF0aCI6Ii8xMzY2NzY1NjQvNDcxNTEzOTY3LWE1MWMzN2VkLTA1NGYtNDFlNy1hNjZhLTNjNTRiODUxNTRmMS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUxMDI3JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MTAyN1QxNTA2NTVaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1hMmQwNzc4MWM5YzhjYTk5NGZhM2EzYzU3NTJkYjdlNGNhY2Q3NDA2OTgyMzZhODA2YTliM2NlMDBjZGY3YzEwJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.m62D7b32Un5TOJ92KVAwliqYls_Bx0GrYMWxaOuTL9A">
> version: <img alt="Image" width="470" height="267" src="https://private-user-images.githubusercontent.com/136676564/471513788-9b0691d0-9200-4478-84bf-876ff79d0c15.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NjE1Nzc5MTUsIm5iZiI6MTc2MTU3NzYxNSwicGF0aCI6Ii8xMzY2NzY1NjQvNDcxNTEzNzg4LTliMDY5MWQwLTkyMDAtNDQ3OC04NGJmLTg3NmZmNzlkMGMxNS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUxMDI3JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MTAyN1QxNTA2NTVaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1iNDA5YmQ2MWE5NWJjMGUwYjFkODMxNDQ4MjFmN2ExMTU3ZGQ1ZTJlMTVjMWUxODg4MjE3NWM4ZDg0ZWM3NjU3JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.HbUcaaah_JHjOblqiXvV1-fQANGHrGgyA6USIBbtCiI">

Unable to install the beta version.. can y'all just release the feature to the live version? It's been 1 year since an official release.

Getting this error when trying to install

App not installed as package appears to be invalid.
